components/ws-daemon/pkg/container/containerd.go

```diff
@@ -306,6 +306,7 @@ func (s *Containerd) handleNewTask(cid string, rootfs []*types.Mount, pid uint32
 func (s *Containerd) WaitForContainer(ctx context.Context, workspaceInstanceID string) (cid ID, err error) {
 	//nolint:ineffassign
 	span, ctx := opentracing.StartSpanFromContext(ctx, "WaitForContainer")
+	span.LogKV("workspaceInstanceID", workspaceInstanceID)
```

Comment:
Can we consider use this function `tracing.ApplyOWI(span, log.OWI("", "", workspaceInstanceID)`?

Reply:
In any other place, I would say yes, but here we only have one variable. Keeping